src/main/java/net/masterthought/cucumber/generators/AbstractPage.java

```diff
@@ -84,6 +87,11 @@ private Properties buildProperties() {
     }
 
     private void buildGeneralParameters() {
+        // to escape html and xml
+        EventCartridge ec = new EventCartridge();
```

Comment:
That would put the EventCartridge and EscapeHtmlReference in a scope that is greater then strictly needed. Neither is expensive to create nor created often so I don't see a good performance reason either.